src/node.h

```diff
@@ -1404,6 +1404,11 @@ NODE_EXTERN void RequestInterrupt(Environment* env,
  * I/O from native code. */
 NODE_EXTERN async_id AsyncHooksGetExecutionAsyncId(v8::Isolate* isolate);
 
+/* Returns the id of the specified execution context. If the return value is
```

Comment:
The `execution context` here does not refer to the `v8::Context`. This function still returns the id of the "current execution context".

```suggestion
/* Returns the id of the current execution context. If the return value is
```